libs/ui-toolkit/arrow-variation/src/lib/arrow-variation.component.ts

```diff
@@ -66,7 +70,12 @@ export class ArrowVariationComponent {
    *
    * @param n number to check
    */
+
   public updateArrow(n: number | '') {
+     if (this.pdfPrint) {
```

Comment:
I think that we can return here, arrowColor and arrowDirection is unused in PDF mode